Makefile

```diff
@@ -265,6 +265,10 @@ CRDOC ?= $(LOCALBIN)/crdoc
 tools:
 	@./hack/tools.sh
 
+.PHONY: kubectl
+kubectl: ## Download kubectl locally if necessary.
+	@./hack/tools.sh $@
```

Comment:
This should be `@./hack/tools.sh kubectl`

Reply:
no .. that works